### ISSUE TITLE
Ubuntu 10 32 bits: error while loading shared libraries: libcdio.so.7

### ISSUE THREAD

**Issue by bass000** [akezeke/spotyxbmc#26]
Hi,

i tried to install the binaries distribution, install.sh went ok.

But when launching xbmc I've got the error:
error while loading shared libraries: libcdio.so.7

In fact the libcdio I have are:

$ ll /usr/lib/libcdio*
-rw-r--r-- 1 root root 166720 2009-11-05 12:14 /usr/lib/libcdio.a
lrwxrwxrwx 1 root root     21 2010-06-16 21:41 /usr/lib/libcdio_cdda.so.0 -> libcdio_cdda.so.0.0.5
-rw-r--r-- 1 root root  26132 2009-11-05 12:14 /usr/lib/libcdio_cdda.so.0.0.5
-rw-r--r-- 1 root root    817 2009-11-05 12:14 /usr/lib/libcdio.la
lrwxrwxrwx 1 root root     25 2010-06-16 21:41 /usr/lib/libcdio_paranoia.so.0 -> libcdio_paranoia.so.0.0.3
-rw-r--r-- 1 root root  26092 2009-11-05 12:14 /usr/lib/libcdio_paranoia.so.0.0.3
lrwxrwxrwx 1 root root     17 2010-06-19 23:36 /usr/lib/libcdio.so -> libcdio.so.10.0.0
lrwxrwxrwx 1 root root     17 2010-06-16 21:41 /usr/lib/libcdio.so.10 -> libcdio.so.10.0.0
-rw-r--r-- 1 root root 112900 2009-11-05 12:14 /usr/lib/libcdio.so.10.0.0

Any ideas ?

**Comment by v21** [comment]
I have this same issue - first with libfaad (on a fresh new Ubuntu it will provide only libfaad.so.2 , but xbmc now demands libfaad.so.0 ) - now facing the same issue with libcdio

**Comment by akezeke** [comment]
GNU Compact Disc Input and Control Library (libcdio) has nothing to do with the integration of spotify, please ask your XBMC questions at the XBMC forums.

**Comment by nvaken** [comment]
I have exactly the same problem after using the install.sh as described in the README. Changing the oldxbmc.bin back to xbmc.bin recovered XBMC, but, as would be expected, fail the install of spotyxbmc...

Trying to:
sudo ln -s /usr/lib/libfaad.so /usr/lib/libfaad.so.0

Solved the error for libfaad

But then a same error popped up for libcdio, trying to solve this in the same manner:
sudo ln -s /usr/lib/libcdio.so /usr/lib/libcdio.so.7

This resulted in a version conflict. 

Downloaded the libcdio, installed, error disapeared. After that, XBMC tried to boot itself but with massive errors and only a black window. Errors:

$ xbmc
X Error of failed request:  BadValue (integer parameter out of range for operation)
  Major opcode of failed request:  53 (X_CreatePixmap)
  Value in failed request:  0x0
  Serial number of failed request:  112
  Current serial number in output stream:  119

I've given up after that, my knowledge stopped at this point. I've recovered xbmc.bin (as described above). Would love to know what is going on?

edit:
Some system specs:
ALSA: Advanced Linux Sound Architecture Driver Version 1.0.23.
Linux #44-Ubuntu desktop (10.10) SMP Fri Jan 21 17:40:48 UTC 2011 i686 GNU/Linux
